tests/data_io/test_csv.py

```diff
@@ -59,17 +59,17 @@
 "id","first_name","nickname","last_name","ssn"
 """
 
-IN_MEM_TAXONOMY_CSV = io.StringIO(
-    "Dairy\n"
-    "Dairy,Cheese\n"
-    "Dairy,Milk\n"
-    "Meat\n"
-    "Meat,Beef\n"
-    "Meat,Chicken\n"
-    "Meat,Chicken,bone-in\n"
-    "Meat,Chicken,boneless\n"
-    "Vegetables\n"
-)
+TAXONOMY_DATA = [
```

Comment:
When this is a stringIO object, it can only be read once. This was causing tests to fail since only the first test to reference it could actually get any data.

Tests weren't failing before because of a typo in a test name (`text_overwrite_file` instead of `test_overwrite_file`) which meant that this data was only being accessed once.

Reply:
Nice find!